src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,61 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.ResolverStyle;
+
+/**
+ * Tasks.Deadline represents a Tasks.Task with a deadline represented as a single LocalDateTime object.
+ * Inherits from Tasks.Task.
+ */
+public class Deadline extends Task {
+    static final String DEADLINE_DATETIME_FORMAT = "d/MM/uuuu HHmm";
+
+    protected LocalDateTime by;
+
+    /**
+     * Overloaded constructor for Deadline.
+     *
+     * @param taskDescription Description of Deadline.
+     * @param by              LocalDateTime representation of the deadline.
+     */
+    public Deadline(String taskDescription, LocalDateTime by) {
+        super(taskDescription);
+        this.by = by;
+    }
+
+    /**
+     * Overloaded constructor for Deadline.
+     *
+     * @param taskDescription Description of Deadline.
+     * @param isComplete      Whether the Deadline is completed.
+     * @param by              LocalDateTime representation of the deadline.
+     */
+    public Deadline(String taskDescription, boolean isComplete, LocalDateTime by) {
+        super(taskDescription);
+        this.setIsComplete(isComplete);
```

Comment:
Rather than creating a task with its isComplete = false, wouldn't it be better to make all tasks created to have their IsComplete always initially false?